content/training/Walk-through/template_setup.Rmd

```diff
@@ -159,7 +171,7 @@ reporting_week <- aweek::as.aweek("2017-W16") # Set reporting_week as last compl
 If you want to use the automatically-updated date from your computer, use `Sys.Date()` like this: `date2week(Sys.Date())`
 {{% /notice %}}
 
-**Lastly, the labels that will be used in epidemic curve plots are defined**. These should not need any adjustment. You may notice that the subtitle label uses the special term "%s" as a placeholder - the reporting_week defined in the previous chunk will show in its place.  
+**Lastly, the labels that will be used in epidemic curve plots are defined**. These should not need any adjustment. You may notice that the subtitle label uses the special term `%s` as a placeholder - the reporting_week defined in the previous chunk will show in its place.  
```

Comment:
d'oh! That was something I should have changed!

```suggestion
**Lastly, the labels that will be used in epidemic curve plots are defined**. These should not need any adjustment. You may notice that the subtitle label uses `reporting_week` in curly braces---this means that the value of `reporting_week` defined in the previous chunk will show in its place.  
```

The epicurve labels for the templates should be generated sing `str_glue` from stringr like this:

```r
subtitle = str_glue("Source: MSF data from {reporting_week}")
```